### ISSUE TITLE
Wanted: Cost to run this API

### ISSUE THREAD

**Issue by Brian-Bartels** [meetDeveloper/freeDictionaryAPI#140]
I am working on a small project that utilizes this API. I've hit your endpoint maybe 25 times. Anyway before I share my project with friends and family I would love to know the monthly costs for this endpoint as well as cost per hit. So that if my site get's any appreciable about of usage I can donate appropriately

**Comment by leifadev** [comment]
I don't believe there isn't too much cost at all, they said they are sourcing now exclusively from Wikitonary. I forgot where I saw this but was definitely on the github or maybe there website.

**Comment by akbayt** [comment]
@leifadev I guess the cost mentioned here is not the cost for a dictionary API or such. The data source was google and to avoid legal issues it was moved to Wiktionary. What costs here is the server. Correct me if I am wrong. Did anyone mention that the developer is paying for a different service?

**Comment by leifadev** [comment]
@akbayt I have no idea, hopefully the transition will complete sooner or later
